### ISSUE TITLE
Metaphase I oocyte meiotic arrest

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6469]
**Preferred term label:**
Metaphase I oocyte meiotic arrest

**Synonyms**
Oocyte meiotic arrest at metaphase I

**Definition (free text, please give PubMed ID)**
Failure of oocytes to progress beyond metaphase I of meiosis.
Comment:  In prophase of meiosis I, genetic information is exchanged between homologous maternal and paternal chromosomes through the process of homologous recombination, which ensures that chromosomes segregate accurately when the meiotic cell divides. After the diplotene stage of prophase I, oocytes are arrested as germinal vesicle (GV) oocytes. In puberty, a surge of luteinizing hormone triggers meiotic resumption, which consists of chromatin condensation, breakdown of the nuclear envelope, spindle formation, and chromosome alignment, and meiosis I is completed upon extrusion of the first polar body (PB1). Therefore, defects in the process of meiosis may cause oocyte maturation arrest.
PMID:32473092

**Parent term (use hpo.jax.org/app)**

Revise hierarchy, considering existing term Spermatogenesis maturation arrest

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 619011 
PMID: 32473092  3/4

Note also 1/4 Zygotic cleavage failure (see #6468 )

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
This term already existed...https://hpo.jax.org/app/browse/term/HP:0031516 improving definition
